app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb

```diff
@@ -88,11 +88,11 @@ fi
      end
    elsif @subman_setup_scenario == 'provisioning'
      if plugin_present?('katello')
-       server_hostname = @host.content_source.rhsm_url.host
+       server_hostname = @host.content_source.registration_url.host
        server_port = @host.content_source.rhsm_url.port
        server_prefix = @host.content_source.rhsm_url.path
```

Comment:
```suggestion
       server_port = @host.content_source.registration_url.port
       server_prefix = @host.content_source.registration_url.path
```

Reply:
The `rhsm_url` path is `"/rhsm"` which isn't appended to the end of the registration url.
```
[3] pry(main)> cs.rhsm_url
=> #<URI::HTTPS https://centos9-stream.fedora.example.com/rhsm>
[4] pry(main)> cs.registration_url
=> #<URI::HTTPS https://centos9-stream.fedora.example.com:9090>
```

Reply:
ACK, make sense.